csrc/scheduler/utils.h

```diff
@@ -197,8 +197,13 @@ bool canProjectToPersistentProducer(
     const std::vector<TensorView*>& producers,
     const std::unordered_set<TensorView*>& persistent_buffer_set);
 
-// Returns all broadcast tvs.
-std::vector<TensorView*> getBroadcastTvs(const std::vector<Val*>& tvs);
+// Returns all broadcast tvs the persistent buffer tv can be projected to.
```

Comment:
Please give more information rather than just what it does. Why is this done? Why broadcast tensors?

Reply:
added more comments.